tests/unit/test_update_movie.py

```diff
@@ -1 +1,17 @@
-# TODO: Feature 5
+from src.models.movie import Movie
+
+
+def test_update_movie():
+    # Setup: Create a repository and add a test movie
+  def test_movie_model():
```

Comment:
remove this, i dont think the test will actually reach this since it's a method in a method and the inner method is not called. 

- Keep test_update_movie
- Remove test_movie_model

Reply:
just fixed update test